src/utils/EDA.py

```diff
@@ -0,0 +1,177 @@
+# Libraries 
+import pandas as pd
+import numpy as np 
+
+# Plotting
+import matplotlib.pyplot as plt
+import seaborn as sns
+
+
+def plt_missing_data(data, title):
+    '''
+    Plots a bar chart showing the percentage of missing values for each column.
+    
+    Args:
+        data (pandas.DataFrame): The DataFrame to plot.
+        str (str): A string to include in the chart title indicating the intended cleaning action.
+
+    Returns:
+        None. Displays a bar chart using matplotlib.
+    '''
+    # Calculate the percentage of missing values for each column
+    missing_percentages = data.isnull().mean() * 100
+
+    # Filter the columns that have missing values
+    missing_columns = missing_percentages[missing_percentages > 0]
+
+    # Sort the columns by percentage of missing values
+    missing_columns = missing_columns.sort_values(ascending=False)
+
+    # Plot the percentage of missing values for each column
+    plt.bar(missing_columns.index, missing_columns.values)
+    plt.title(title)
+    plt.xlabel('Column Name')
+    plt.ylabel('Percentage of Missing Values')
+    plt.xticks(rotation=90)
+    plt.show()
```

Comment:
It is better to verify whether or not there is data to plot.

Reply:
it will display an empty plot if there's no data to plot